.circleci/config.yml

```diff
@@ -8,6 +8,11 @@ defaults: &defaults
     - image: cimg/node:18.16.0
     - image: cimg/postgres:14.1
     - image: redis:7.0.12-alpine
+    - image: rabbitmq:3.13.0-rc.1-alpine
```

Comment:
I wouldn't use release candidate version of images - I'd recommend switching to `3.12.8-alpine`